README.md

```diff
@@ -102,6 +102,8 @@ dependencies {
   - Remove setElementValue helper. [#1577](https://github.com/appium/java-client/pull/1577)
   - Remove selenium package override. [#1555](https://github.com/appium/java-client/pull/1555)
   - remove redundant exclusion of Gradle task signMavenJavaPublication. [#1568](https://github.com/appium/java-client/pull/1568)
+- **[BUG FIX]
+  - Filter out proxyClassLookup method from Proxy class (for Java 16+) in AppiumByBuilder. [#1575](https://github.com/appium/java-client/issues/1575)
```

Comment:
I don't think it should be in `8.0.0-beta` since it's released

Reply:
8.0.1-beta? Should I bump the version?